pages/[[...slug]].js

```diff
@@ -1,27 +1,37 @@
 import DotCMSPage from '../components/layout/DotCMSPage';
 import Error from '../components/layout/Error';
-import { getPage, getNav, getPathsArray } from '../utilities/dotcms';
+import {
+    getPage,
+    getNav,
+    getPathsArray,
+    getLanguages,
+    getLanguagesProps
+} from '../utilities/dotcms';
 import getPageList from '../utilities/dotcms/getPageList';
 
-export default function Page({ pageRender, nav, error }) {
+export default function Page({ pageRender, nav, error, languageProps }) {
     if (error) {
         return <Error statusCode={error.statusCode} message={error.message} />;
     }
-    return <DotCMSPage pageRender={pageRender} nav={nav} />;
+    return <DotCMSPage pageRender={pageRender} nav={nav} languageProps={languageProps} />;
```

Comment:
Why are we passing all the `languageProps` if inside (client) for what I can tell we are only using the language list?

Reply:
Because `languageProps` returns an object with 3 or 4 properties and we need some of them in the client.